Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedGraphicsLayer.cpp

```diff
@@ -1221,6 +1224,8 @@ void CoordinatedGraphicsLayer::updateContentBuffers()
     if (!dirtyTiles.isEmpty()) {
         bool didUpdateTiles = false;
 
+        WTFBeginSignpost(this, PaintTiles, "dirty tiles: %lu", dirtyTiles.size());
```

Comment:
I would also add signposts in the inner loop over the dirtyTiles including information such as which tiles is updated: tile 1/N, tileRect + dirtyRect + tileID -- this allows for visual inspection when tiles first appear, are re-used or disappear. Also helpful to track the dirtyRect in sysprof.